packages/medusa/src/api/routes/admin/orders/create-swap.ts

```diff
@@ -98,7 +99,9 @@ import { validator } from "../../../../utils/validator"
 export default async (req, res) => {
   const { id } = req.params
 
-  const validated = await validator(AdminPostOrdersOrderSwapsReq, req.body)
+  const validated = req.validatedBody
+
+  const include_returnable_items = req.includes?.returnable_items
```

Comment:
thought(non-blocking): Instead of adding clutter to our handlers, could we pass `...req.includes` to the retrieve options?

Reply:
I think so, but we can imagine that includes is not necessarily link to the totalFieldsConfig and can contains something else. So I would go for a more verbose but more clear (and people knows what to expect) instead. wdyt?

Reply:
One could argue that having a dedicated config for totals is not ideal. This could be a `context` instead.

That said, it's a tradeoff between verbosity and flexibility, and I am not sure which one is more appropriate. What's your gut-feeling?

Reply:
I agree, I will just change the name of the parameter to context and add an includes property which would take the includes object. 👍